### DISCUSSION TITLE
Double range slider (new feature?)

### DISCUSSION THREAD

**Discussion by Davidm16** [flet-dev/flet#1691] [Q&A]
### Question

I need a widget that can change a minimum and maximum range for my project but I haven’t found anything related on the documentation. Is it possible to create a double range slider like this?

![image](https://github.com/flet-dev/flet/assets/16451684/392ffda9-b60f-4247-b929-60cc73a034d0)

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by FeodorFitsner** [comment]
You can build your own control. This could be an inspiration: https://github.com/flet-dev/flet-contrib/tree/main/flet_contrib/flexible_slider

**Reply by Davidm16** [reply]
It’s too much mess and work to have to design a new widget if flutter is already native

https://api.flutter.dev/flutter/material/RangeSlider-class.html

**Reply by FeodorFitsner** [reply]
Oh, nice! Please create a new issue then to implement that widget in Flet.

**Reply by FeodorFitsner** [reply]
https://github.com/flet-dev/flet/issues/1712
